CMakeCommon.cmake

```
@@ -17,8 +17,10 @@ function (SetCompilerOptions target acVersion)
 
     if (${acVersion} LESS 27)
         target_compile_features (${target} PUBLIC cxx_std_14)
-    else ()
+    elseif (${acVersion} LESS 29)
         target_compile_features (${target} PUBLIC cxx_std_17)
+	else ()
+		target_compile_features (${target} PUBLIC cxx_std_20)
```

```suggestion
    else ()
        target_compile_features (${target} PUBLIC cxx_std_20)
```

This project uses 4 spaces as indentation, let's keep it consistent.